src/mscorlib/src/System/Runtime/InteropServices/NonPortable.cs

```diff
@@ -39,6 +39,12 @@ public static Object BindToMoniker(String monikerName)
         }
         
         [System.Security.SecurityCriticalAttribute]
+        public static void CleanupUnusedObjectsInCurrentContext()
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
Would it make sense to have this be a no-op instead? There would be nothing to clean up so it would effectively do nothing. I was thinking of doing the same for `Thread.DisableComObjectEagerCleanup`.